### PR TITLE
Update output terminology

### PR DESCRIPTION
**Pull request by sundowndev**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | no
| 🚀 New feature?   | no
| ⚠ Deprecations?   | no
| ❌ BC Break       | **yes**
| 🔗 Related issues | #372
| ❓ Documentation  | yes 

## Description

We should update the wording of the output to be consistent with the summary.

```
Found changed resources:
  - 09721498761984 (aws_s3_bucket):
    ~ Versioning.0.Enabled: false => true
Found 1 resource(s)
 - 100% coverage
 - 1 covered by IaC
 - 0 not covered by IaC
 - 0 missing on cloud provider
 - 1/1 changed outside of IaC
```

### CODE REVIEW

**Review by moadibfr — APPROVED** [review]

**Review by eliecharra — CHANGES_REQUESTED** [review]
As requested in the issue, we need to update JSON output too :

> JSON:
> Change deleted to missing

**Comment by sundowndev** [comment]
> As requested in the issue, we need to update JSON output too :
> 
> > JSON:
> > Change deleted to missing

@eliecharra done, waiting for another review

**Review by eliecharra — APPROVED** [review]

**Pull request merged**
